molecule/driver/lxd.py

```diff
@@ -38,6 +38,17 @@ class LXD(base.Base):
 
         driver:
           name: lxd
+        platforms:
+          - name: instance
+            url: https://127.0.0.1:8443
+            cert_file: "/root/.config/lxc/client.crt"
+            key_file: "/root/.config/lxc/client.key"
+            trust_password: password
+            mode: pull|local
+            server: https://images.linuxcontainers.org
+            alias: ubuntu/xenial/amd64
+            profiles:
+              - default
```

Comment:
Since you have implemented this API for molecule.yml we need to validate the schema.  Can you update the v2 schema similar to what we did with dockder and vagrant.

https://github.com/metacloud/molecule/blob/master/molecule/model/schema_v2.py#L596
https://github.com/metacloud/molecule/blob/master/molecule/model/schema_v2.py#L862

And we will need to update tests with working and failing tests:

https://github.com/metacloud/molecule/blob/master/test/unit/model/v2/test_platforms_section.py